statics/js/components/topology.js

```diff
@@ -314,21 +328,89 @@ var TopologyComponent = {
 
     topologyFilterClear: function () {
       this.topologyFilter = '';
+      this.topologyFilterQuery();
      },
 
     topologyHighlightClear: function () {
       this.topologyHighlight = '';
+      this.topologyHighlightQuery();
      },
 
     endsWith: function (str, suffix) {
        return str.indexOf(suffix, str.length - suffix.length) !== -1;
     },
 
+    setFilterFromConfig: function() {
+      return $.when(this.$getConfigValue('analyzer.topology_filter'))
+       .then(function(filter) {
+         var options = $("#topology-filter-list");
+         $.each(filter, function(key, value) {
```

Comment:
if there is no filter in config file, getting null reference error in jquery lib

Reply:
Thanks for reporting this issue, fixed